api/controllers/deployments.js

```diff
@@ -21,10 +21,12 @@ function getDeployments(req, res, next) {
     offset: req.swagger.params.offset.value,
     order: [ ["createdAt", "DESC"] ]
   }).then(function(deployments) {
-    for(var i = 0; i < deployments.length; i++) {
-      var deployment = deployments[i];
-      deleteNullValues(deployment.dataValues);
-    }
+    deployments.map(function(deployment) {
```

Comment:
Does `map` act any differently than `forEach` given that you aren't assigning the return values to a new array?